alembic.ini

```diff
@@ -0,0 +1,66 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = migrations
+
+# template used to generate migration file names; The default value is %%(rev)s_%%(slug)s
+file_template = %%(year)d_%%(month).2d_%%(day).2d_%%(hour).2d%%(minute).2d-%%(rev)s_%%(slug)s
+
+# sys.path path, will be prepended to sys.path if present.
+# defaults to the current working directory.
+prepend_sys_path = .
+
+# timezone to use when rendering the date within the migration file
+# as well as the filename.
+# If specified, requires the python-dateutil library that can be
+# installed by adding `alembic[tz]` to the pip requirements
+# string value is passed to dateutil.tz.gettz()
+# leave blank for localtime
+# GMT+0 means UTC timezone
+timezone = GMT+0
+
+# max length of characters to apply to the "slug" field
+truncate_slug_length = 80
+
+# version_path_separator = :
+# version_path_separator = ;
+# version_path_separator = space
+version_path_separator = os  # Use os.pathsep. Default configuration used for new projects.
+
+sqlalchemy.url = postgresql://postgres:@localhost:5432/postgres
```

Comment:
Here we need to solve same problem which is described [here](https://github.com/EugeneTurkin/forum123/pull/17/files#r1066802661). We will discuss it together when you fix this problem at the place where link points to.